api/CSSStyleValue.json

```diff
@@ -142,6 +142,53 @@
             "deprecated": false
           }
         }
+      },
+      "toString": {
```

Comment:
https://mdn-bcd-collector.appspot.com/tests/api/CSSStyleValue/toString has the problem described in https://github.com/foolip/mdn-bcd-collector/issues/841.

I think this data is correct, but if `toString` had been added later, the test wouldn't have caught that. Can you double check this one?

Reply:
Can confirm this one works in Chrome 66!

Reply:
Thanks!